JL.Windows/GUI/PopupWindow.xaml.cs

```diff
@@ -695,6 +695,31 @@ private StackPanel PrepareResultStackPanel(LookupResult result, int index, int r
             audioButton.Click += AudioButton_Click;
 
             _ = top.Children.Add(audioButton);
+
+            if (!configManager.MineToFileInsteadOfAnki)
```

Comment:
We can check whether we need to call `CheckDuplicate` at all here, like:
```C#
if (coreConfigManager.CheckForDuplicateCards
    && !configManager.MineToFileInsteadOfAnki
    && !coreConfigManager.AnkiIntegration)
```
The following check can be removed from `CheckDuplicate`:

```C#
if (!coreConfigManager.AnkiIntegration || !coreConfigManager.CheckForDuplicateCards)
{
    return false;
}
```